src/main/java/org/example/bookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package org.example.bookstore.exception;
+
+public class RegistrationException extends RuntimeException {
```

Comment:
Actually task requirements imply that it should be a checked exception:
> Add AuthenticationController with the following methods:
public UserResponseDto register(UserRegistrationRequestDto request) throws RegistrationException